src/LeakyBucket/redis.lua

```diff
@@ -0,0 +1,53 @@
+-- Gets keys and arguments from command
+local resourceLastUpdatedKey = KEYS[1]
+local resourceBucketSizeKey = KEYS[2]
+local maxBucketSize = tonumber(ARGV[1])
+local leakRateValue = tonumber(ARGV[2])
+local leakRateDuration = tonumber(ARGV[3])
+local leakFullTime = tonumber(ARGV[4])
+local now = tonumber(ARGV[5])
+local worth = tonumber(ARGV[6])
+
+local ttlSecs = math.ceil(leakFullTime/1000)
+local retryAfter = -1
+
+-- Reads values of 2 keys - last updated and current bucket size against key
```

Comment:
move logic to get last upfated, bucket size and leak to a separate method?

Reply:
Why separate method?

This script is not suppose to increase over time and is well readable(not?) in current form. 

Adding methods will unnecessarily increase call stacks (tiny but unnecessary).